src/System.Runtime.Extensions/tests/System/BitConverter.cs

```diff
@@ -207,7 +209,20 @@ public static void RoundtripString()
             Assert.Equal("12-34-56-78-9A", BitConverter.ToString(bytes));
             Assert.Equal("56-78-9A", BitConverter.ToString(bytes, 2));
             Assert.Equal("56", BitConverter.ToString(bytes, 2, 1));
-            Assert.Equal(String.Empty, BitConverter.ToString(new byte[0]));
+
+            Assert.Same(string.Empty, BitConverter.ToString(new byte[0]));
+            Assert.Same(string.Empty, BitConverter.ToString(new byte[3], 1, 0));
```

Comment:
> just that I'd've written Equal and I'm indirectly wondering what I'm missing.

Equals is fine, it's just not "as strong" as you say.  I think we want to be able to do things in our tests that "lock in" behaviors we want to ensure are there, even if they're not documented or guaranteed behaviors in MSDN documentation.  We have (or will have) perf tests that help us to  track when performance regresses, even though we don't publicly document details of how fast something is or how much it allocates; similarly, we should be able to have tests that help us to track when undocumented/non-guaranteed behaviors change.  If we end up making such a change and the test fails, we can then think through whether it's a change we're willing to accept or not, and either fix the fix or change the test at that point.  Reasonable?